notebooks/book1/02/softmax_plot.ipynb

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        plot_softmax_distribution(T, a, f"softmax_dist_{i+1}_latexified")
</pre><p>Now you can remove <code>_latexified</code>, <code>save_fig</code> will add it.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>